Makefile

```diff
@@ -106,3 +106,8 @@ proto: install-tools ## Generate protobuf files
 .PHONY: bench-kafka
 bench-kafka:
 	go test -count 1 -run BenchmarkCompression -bench=. -benchmem ./services/streammanager/kafka/client
+
+.PHONY: serve-swagger-ui
+serve-swagger-ui: install-tools
```

Comment:
You would need to add `go install github.com/go-swagger/go-swagger/cmd/swagger@latest` inside install-tools.

Reply:
Added.